config/environments/development.rb

```diff
@@ -69,7 +69,7 @@
   # config.i18n.raise_on_missing_translations = true
 
   # Annotate rendered view with file names.
-  # config.action_view.annotate_rendered_view_with_filenames = true
+  config.action_view.annotate_rendered_view_with_filenames = true
```

Comment:
What does this do?

Reply:
It shows you in HTML comments in the development server which views are being rendered.